src/controllers/PostController.js

```diff
@@ -134,7 +134,7 @@ export default class PostController {
   static async getPosts(req, res) {
     const posts = await Post.find({})
       .select('-__v')
-      .populate('author', '-_id -__v -userType -password');
+      .populate('author', 'picture username firstName lastName followerCount followedCount country city ');
 
     return res.status(statusCodes.OK).json({
       status: statusCodes.OK,
```

Comment:
@oesukam We should also enable the user to post as an organization

Reply:
It is still applies. I didn't want to remove the actual user who posted as the organization even if we are not going to display those information @gracelungu